lib/presentation/book/bloc/home_page_bloc.dart

```diff
@@ -8,23 +8,33 @@ import 'package:flutter_bloc/flutter_bloc.dart';
 class HomePageBloc extends Bloc<HomePageEvent, HomePageState> {
   HomePageBloc() : super((InitialState())) {
     on<AddNewBook>(_onBookAdded);
-    on<RefreshBooks>(_onRefreshBooks);
+    on<onNewBookAdded>(_onNewBookAdded);
     on<AddBookImageEvent>(_onAddBookImage);
     on<DeleteBookImage>(_onDeleteBookImage);
+    on<DownloadBooks>(_onDownloadBooks);
   }
 
   Future<void> _onBookAdded(
       AddNewBook event, Emitter<HomePageState> emit) async {
     emit(LoadingState());
 
-    final urlResult =
-        await FirebaseDbManager.internal().AddBookImage(event.imageFile);
+    try {
+      final urlResult =
+          await FirebaseDbManager.internal().uploadBookImage(event.imageFile);
```

Comment:
why internal()? we should call instance...

Reply:
internal method should be private method, not public